etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/CreateRawBaseTables.java

```diff
@@ -278,16 +273,20 @@ public void executeStageLoad(String[] params) {
     }
 
 
+    private String getName(String name){
+        return "SHOW TABLES LIKE '" + name + "'";
+    }
 
     private void checkAndCreateRawTable(String dbName, String tableName, String ddl) {
         try {
             LOGGER.debug("Reading Hive Connection details from Properties File");
             Connection con = getHiveJDBCConnection(dbName);
             Statement stmt = con.createStatement();
-            ResultSet rs = stmt.executeQuery("SHOW TABLES LIKE '" + tableName + "'");
+            String tName = tableName;
+            ResultSet rs = stmt.executeQuery("SHOW TABLES LIKE '" + tName + "'");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) "tableName" is provided externally to the method and not sanitized before use. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2077)